WalletWasabi.Gui/Styles/Styles.xaml

```diff
@@ -55,91 +51,67 @@
   </Style>
-
   <Style Selector="Grid.content">
     <Setter Property="TextBlock.FontSize" Value="14" />
   </Style>
-
   <Style Selector=".hyperLink">
     <Setter Property="TextBlock.Foreground" Value="{DynamicResource HyperLinkForeground}" />
   </Style>
-
   <Style Selector=".errorMessage">
     <Setter Property="TextBlock.Foreground" Value="{DynamicResource ErrorBrush}" />
   </Style>
-
   <Style Selector=".warningMessage">
     <Setter Property="TextBlock.Foreground" Value="{DynamicResource WarningMessageForeground}" />
   </Style>
-
   <Style Selector="Button.activeHyperLink">
     <Setter Property="TextBlock.Foreground" Value="{DynamicResource HyperLinkForeground}" />
     <Setter Property="Cursor" Value="Hand" />
     <Setter Property="BorderThickness" Value="0" />
     <Setter Property="Padding" Value="0" />
     <Setter Property="Background" Value="Transparent" />
   </Style>
-
   <Style Selector="dock|DockToolChrome:active /template/ Grid#PART_Grip">
     <Setter Property="Background" Value="{DynamicResource ThemeBackgroundBrush}" />
   </Style>
-
   <Style Selector="Button.chromeButton">
     <Setter Property="IsVisible" Value="False" />
   </Style>
-
   <Style Selector="ListBox">
     <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderLowBrush}" />
   </Style>
-
   <Style Selector="ListBoxItem">
     <Setter Property="Padding" Value="5 0" />
   </Style>
-
   <Style Selector="TextBox">
     <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderHighBrush}" />
   </Style>
-
   <Style Selector="DataValidationErrors">
     <Setter Property="Template">
       <ControlTemplate>
         <DockPanel LastChildFill="True">
-          <ContentControl DockPanel.Dock="Bottom"
-                        ContentTemplate="{TemplateBinding ErrorTemplate}"
-                        DataContext="{TemplateBinding Owner}"
-                        Content="{Binding (DataValidationErrors.Errors)}"
-                        IsVisible="{Binding (DataValidationErrors.HasErrors)}" />
-          <ContentPresenter Name="PART_ContentPresenter"
-                          Background="{TemplateBinding Background}"
-                          BorderThickness="{TemplateBinding BorderThickness}"
-                          ContentTemplate="{TemplateBinding ContentTemplate}"
-                          Content="{TemplateBinding Content}"
-                          Padding="{TemplateBinding Padding}" />
+          <ContentControl DockPanel.Dock="Bottom" ContentTemplate="{TemplateBinding ErrorTemplate}" DataContext="{TemplateBinding Owner}" Content="{Binding (DataValidationErrors.Errors)}" IsVisible="{Binding (DataValidationErrors.HasErrors)}" />
+          <ContentPresenter Name="PART_ContentPresenter" Background="{TemplateBinding Background}" BorderThickness="{TemplateBinding BorderThickness}" ContentTemplate="{TemplateBinding ContentTemplate}" Content="{TemplateBinding Content}" Padding="{TemplateBinding Padding}" />
```

Comment:
The original was better readable.

Reply:
I agree. Should we put each attribute on a separate line in all `xaml` files? If we do that the files will be long but much more readable.

Reply:
Only when the attributes are too many, like this.